backend/src/Edna/Upload/Service.hs

```diff
@@ -130,11 +133,23 @@ uploadFile' projSqlId@(SqlId proj) methodSqlId@(SqlId method)
       measurementsToSummary fileMeasurements
 
 insertTarget :: Text -> Edna (Text, TargetId)
-insertTarget targetName = (targetName,) . fromSqlSerial . tTargetId <$> LQ.insertTarget targetName
+insertTarget targetName = do
+  (targetName,) . fromSqlSerial . tTargetId <$> LQ.insertTarget targetName
+
+
```

Comment:
Normally there is 1 empty line between functions, 2 empty lines should be used only in special cases and I think this one is not special.